README.md

```diff
@@ -43,6 +43,7 @@ Download a zip of the code via GitHub or PyPi. Then follow the same instructions
 * AAD Username/password - Provide your AAD username and password to Kusto client (**check the notice below**).
 * AAD application - Provide app ID and app secret to Kusto client.
 * AAD code - Provide only your AAD username, and authenticate yourself using a code, generated by ADAL.
+* AZ CLI - Prerequisites include running "az login". May need to also run "az account get-access-token" once.
```

Comment:
+ Az Cli installed Obviously :)

Reply:
Yep! Just in case someone isn't sure about azure-cli or its setup,  I will add a link to the azure-cli repo here as it's better addressed there.